.github/workflows/set-unboud-pipx-variable-test.yml

```diff
@@ -14,20 +14,24 @@ jobs:
             - windows-latest
 
       env:
-        SNOWFLAKE_CONNECTIONS_MYCONNECTION_PASSWORD: ${{ secrets.SNOWFLAKE_PASSWORD }}
-        SNOWFLAKE_CONNECTIONS_MYCONNECTION_ACCOUNT: ${{ secrets.SNOWFLAKE_ACCOUNT }}
         PIPX_BIN_DIR: null
 
       name: Install Snowflake CLI in ${{ matrix.os }}
       steps:
         - uses: actions/checkout@v4
+          with:
+            persist-credentials: false
+
         - name: Set up Snowflake CLI
           uses: ./ 
           id: setup_snowcli
           with:
             default-config-file-path: '.github/workflows/tests-files/config.toml'
 
         - name: Test connection 
+          env:
+            SNOWFLAKE_CONNECTIONS_MYCONNECTION_PASSWORD: 'password'
+            SNOWFLAKE_CONNECTIONS_MYCONNECTION_ACCOUNT: 'account'
```

Comment:
What happened with secret variables?

Reply:
They are not necessary because the pipelines don't tests connection, only installation

Reply:
Yes, it is a better name